### ISSUE TITLE
Multiline string inside lambda expression fails to format

### ISSUE THREAD

**Issue by shivinsky** [fsprojects/fantomas#1299]
Issue created from [fantomas-online](https://fsprojects.github.io/fantomas-tools/#/fantomas/preview?data=N4KABGBEDGD2AmBTSAuKAbRAXMB9MAvADoBORAdhBANoC6YAPgHxgAyAlgM5YB0AtgEMADmAAUAMwCulfAFoWAASKQBpCgCNlVCAEo1lbRGZsuvTOQDmWABaQANOChJx7cok6oo9x5C4AxTnZPcQF0TkQQAF8gA)

#### Code

```fsharp
let _ =
    [] |> List.map (fun _ -> @"a
b"     )
       |> List.length
```
            

#### Error

```fsharp
Fantomas was able to format the code but the result appears to be invalid F# code.
Please open an issue.

Formatted result:

let _ =
    []
    |> List.map
        (fun _ ->
            @"a
b")
    |> List.length

```
            
#### Extra information

- [x] The formatted result breaks by code.
- [ ] The formatted result gives compiler warnings.
- [ ] I or my company would be willing to help fix this.

#### Options

Fantomas Master at 12/17/2020 16:49:57 - a212bf0a78cef0f84d86b95539ed62b15a6245fb

Default Fantomas configuration

**Comment by nojaf** [comment]
Hello, thanks for this report. I still don't really know how to solve this but I wanted to let you know that the problem does not seems to occur when [fsharp_multi_line_lambda_closing_newline=true](https://github.com/fsprojects/fantomas/blob/master/docs/Documentation.md#fsharp_multi_line_lambda_closing_newline).
[Sample](https://fsprojects.github.io/fantomas-tools/#/fantomas/preview?data=N4KABGBEDGD2AmBTSAuKAbRAXMB9MAvADoBORAdhBANoC6pFVYAPgHxgAyAlgM5YB0AWwCGABwaUmVABQAzAK6V8AWlYSpGsAAEikYRIBGujQEp1VNp14DM5AOZYAFpAA04KD2xYu9nqjDU7hCgGpAAJFgAnqLIaJA%2BWK5BVOEAbsLo8rEByUwADG6aUACS5EjkWADKXABeyIWaACy5YLTJAL4NwS3hUTH%2B8RVJoWHpmdmBRQCMXVKQALLCAB7c5IgciPZOw5pTAEx5LW1MnckhcxHR2ZAGsLDoO0xpGVn%2Bk5p7s0%2BViIJccOhYOQAIJYACiZQA8rJVvUWhBZBlPEcOrNzk9Lv04rd7o8UqMXhN4WAAMxfFKVUTCaCIABCiFksBIiAACsISMJBNhECQ8UwsCQsiiTmiepjrjiHuTnuM3sTGuSIJBKdS6QymetYAB3HnQYSeUqpWB67xAvlUAVCjTHKinJjo-F9CV3KVisavNDvDQAVkVUBVNPpjOZAFVRDESHqDeQjSauGa-Yj0MjraizmKnQNJeaZR6ckUAGx%2B5VUwPq5kAYXQ%2Bp4FaBfEF0CwTPNCKRiGFttFI0z2JdOYJss9xIA7MWA2rg4h5r8DDzW2AkympDaIHaqA6leKs-3pYO816pAAOcelycauuA8gLpcd1Mi9M9q473F791EooATlPquBsiwPJ1oIIgLpad4rmm9oZs%2Bfavm6hJytMBTEiWqpBhqPx-ACCbErenZrt2Fy9lA2Zvghw7TDMKETn%2BAEkJh-z3DhRRgfhYDrt0T5YiRu7wUO%2Ba7J8KGlOUWCQuQAAqJCRAA6lw2yJu2bEcWAm5QNusGuiM76IbsZLUWewIkLAijwAAIog6BcH8dGgYK4FMKu7GERixGDIkZH8YeTBTAqKGLEsxSyBJjibGCyaIJUjhMlgcnwAp8qHPeXaPkRMFQAkA46RRuy%2Bv5yylLIXBLJCEbCM2JBgksojMjwPDxtefreklEEPlBXHXJlnkHsSUxFvlSwAEqIHAJDwHFCVFI0LWOZBG7Qdx7lZeRAkaFMY4DcNo3wAAcvIghziQ0LFABgh%2BH6UzKS5jrpZAgjyOg3hWWsABiTIiFgdESc%2B3UfrsJ4oVtTLwPMD1PT4iBvSQH22cW0COOy1J0bgWpcPFzhXalrm3V1fE9dM34DUZHKRJCJDcHwE3OH602Y%2B1aWLbj2krd5VAHMWAXE8IpPk9Ye0HTyx2nedvV0-NHUDPdj1cM9kPveVX0-Xjf0aHsVFFJAXM8xTWCg9LstQzD85wwjHJNjyKNo9sYucQznVDL9umq0JGsBQAaoStI%2BPAPh2FTC608lBFYzdjMO8rTtSHs%2Bmu8sL2KE2DVe2Uvv%2BzTM1UE5Klqb0OPh8zXnEnsfmx0sZmwFgADi2BVTViB1Q1afEs1NuqQtzpwQX%2BMfHlrtgzLEO0oC0AANa0mbI-YDw4mVJyiCXvtjW4UpQfOSHW5uaREc5ar-UaztiBarL9JYDqmzfTEFlFeQ8kNcCZQzgLJAi0UeGr9n7cvlpFzZatUcbRrAA0ogRAoggohU2KUWeXJYQ3hXq1FK9NsZhw8tvP%2BTA9gA1LuFP4PBHBNymhnCAWdrob1ulvLuKso6Ew1tUewmAjJ2H2psWKiADDzAQHCV%2B8DZptXFnbL%2By1C5FBJMhDWwIrJ2HIPHcgicgTVCkeVeQzIJKwBEiw8qDU4ERVbjnDSPFO4-xZsSEk6tNCa0ejycgmjUiansI-Q6V8fC33rNo5cvDEH8OQR3b%2BGJf6swgCSF25i9beFhBwTkBh4DCCrLAeq9gD5HwhnZK0CDg5INDj4oR3cNAkhjuYsyvBhAGEwDg3gjhKiRAqMsNxDlM5zVtt4gYmx4DQlhJUKImBslUKYCSEu5iIStJhMkuGJB0CyEgLoz%2BmlumR16b3cxwDQHqIqKUcewhZHU2Xjo9%2BpD1Kb14pQuZVASR73MUPDZI9YQ8CMiZMoB8%2BCIBBv3WWtdar1VcX6Viuz177PIYcoxwjNAkgAec9k6FmSFJ4NAEg1kfDlSeSGG%2BQILLQGrByU0S9uE7LSWvDJZDFoUMBTkqQJIsHmI6bCpsHCkC1OUu4WgDR4g8BevVfweF2hAA).

**Comment by nojaf** [comment]
Fix available in [v5.2.0-beta-001](https://www.nuget.org/packages/fantomas/5.2.0-beta-001)
